common/sgx/verifier.c

```diff
@@ -250,6 +256,16 @@ static oe_result_t _fill_with_known_claims(
         sgx_report_body = &((sgx_quote_t*)report_body)->report_body;
         OE_CHECK(
             oe_parse_sgx_report_body(sgx_report_body, true, &parsed_report));
+
+        sgx_quote = (sgx_quote_t*)report_body;
+        pcesvn = (uint16_t) & (sgx_quote)->pce_svn;
+        memcpy(qeid, &(sgx_quote)->user_data, sizeof(sgx_quote->user_data));
+        oe_parse_tcb_info_json(
+            sgx_endorsements->items[OE_SGX_ENDORSEMENT_FIELD_TCB_INFO].data,
+            sgx_endorsements->items[OE_SGX_ENDORSEMENT_FIELD_TCB_INFO].size,
+            &local_platform_tcb_level,
+            &parsed_tcb_info);
+        memcpy(fmspc, parsed_tcb_info.fmspc, sizeof(parsed_tcb_info.fmspc));
```

Comment:
Can we move these to  block of format_type != SGX_FORMAT_TYPE_LOCAL, as these are only valid when the evidence is a quote for remote attestation